overlord/snapstate/snapstate_update_test.go

```diff
@@ -12604,32 +12610,34 @@ func (s *snapmgrTestSuite) TestOldStyleAutoRefreshSplitEssentialWithSharedBase(c
 	s.settle(c)
 
 	checkRerefresh := true
-	s.checkSplitRefreshWithSharedBase(c, chg, checkRerefresh)
+	mockRestartFunc := func() { s.mockRestartAndSettle(c, chg) }
+	s.checkSplitRefreshWithSharedBase(c, chg, checkRerefresh, mockRestartFunc)
 }
 
-func (s *snapmgrTestSuite) checkSplitRefreshWithSharedBase(c *C, chg *state.Change, checkRerefresh bool) {
-	for _, snap := range []string{"snapd", "some-base", "some-base-snap"} {
+func (s *snapmgrTestSuite) checkSplitRefreshWithSharedBase(c *C, chg *state.Change, checkRerefresh bool, mockRestart func()) {
```

Comment:
No, the call was pulled out so this helper could be used in a test belonging to a different suite. Since that was replaced in the previous PR, this can go. Thanks!